grpcgcp/gcp_balancer.go

```diff
@@ -277,9 +297,7 @@ func (gb *gcpBalancer) unbindSubConn(boundKey string) {
 	boundSC, ok := gb.affinityMap[boundKey]
 	if ok {
 		gb.scRefs[boundSC].affinityDecr()
-		if gb.scRefs[boundSC].getAffinityCnt() <= 0 {
```

Comment:
Oh, right. This is a small bugfix. When unbinding an affinity key we can remove it from the map immediately rather than waiting for all keys to be unbound on the subconn.